test/common/bond_test.js

```diff
@@ -48,8 +48,20 @@ test('determineBondProvider should return the default BondProvider if the URL ho
     t.is(determineBondProvider('drs://some-host/anything'), BondProviders.default);
 });
 
-test('determineBondProvider should not return a provider if the URL host is JDR"', (t) => {
-    t.falsy(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'));
+test('determineBondProvider should return the "jade-data-repo" as the provider for JDR dev host"', (t) => {
+    t.is(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'), BondProviders.JADE_DATA_REPO);
+});
+
+test('determineBondProvider should return the "jade-data-repo" as the provider for JDR staging host"', (t) => {
+    t.is(determineBondProvider('drs://data.staging.envs-terra.bio/identifier'), BondProviders.JADE_DATA_REPO);
+});
+
+test('determineBondProvider should return the "jade-data-repo" as the provider for JDR alpha host"', (t) => {
+    t.is(determineBondProvider('drs://data.alpha.envs-terra.bio/identifier'), BondProviders.JADE_DATA_REPO);
+});
```

Comment:
Since Jade is going to keep generating hostnames, I think only one of these is needed. That or see slack for the current list, and expect to submit future PRs when they keep adding hosts.